db/migrations/versions/013_add_programme_junction_to_funding_and_output_data_tables.py

```diff
@@ -0,0 +1,40 @@
+"""Add programme_junction_id to output_data and funding
+
+Revision ID: 013_add_programme_junction_to_tables
+Revises: 012_event_data_to_jsonb
+Create Date: 2024-02-26 14:37:37.825821
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "013_add_programme_junction_to_tables"
+down_revision = "012_event_data_to_jsonb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("output_data", sa.Column("programme_junction_id", sa.GUID(), nullable=False))
```

Comment:
also, as a side note - this line would never work on a pre-populated table, since you can never just add a non-nullable table (as all existing lines would instantly break this constraint). The usual work around is to update this script to first of all make the column nullable, then migrate in the correct data, then update the column to be non-nullable. All in one atomic transaction.

Same goes for the downgrade, you need to reverse the data migration done above or the DB will not downgrade (as running downgrade will hit errors and rollback)